src/lib/abi/GaugeController.json

```diff
@@ -290,23 +290,6 @@
       }
     ]
   },
-  {
```

Comment:
This needs to be removed because Ether's cant deal with duplicate functions in ABI's